src/NHibernate/Multi/QueryBatchItemBase.cs

```diff
@@ -31,7 +32,7 @@ protected class QueryInfo : ICachingInformation, ICachingInformationWithFetches
 			/// <summary>
 			/// The query loader.
 			/// </summary>
-			public Loader.Loader Loader { get; set; }
+			public ILoader Loader { get; set; }
```

Comment:
This change here needs also to be avoided.

Reply:
Type of property reverted and additional constructor removed again.